runtime/zeitgeist/src/lib.rs

```diff
@@ -93,8 +93,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("zeitgeist"),
     impl_name: create_runtime_str!("zeitgeist"),
     authoring_version: 1,
-    spec_version: 55,
-    impl_version: 1,
+    spec_version: 56,
+    impl_version: 0,
```

Comment:
Why you did change the `impl_version`? Can't we just leave it as the default, which is one?

Reply:
Oh I didn't read `This number can be reverted to 0 after a [spec_version](https://paritytech.github.io/polkadot-sdk/master/cumulus_test_client/sc_executor/struct.RuntimeVersion.html#structfield.spec_version) bump.`